pkg/pkg/pid.go

```diff
@@ -14,15 +15,19 @@ type PID struct {
 }
 
 func (d PID) String() string {
-	return fmt.Sprintf("%s:%s:%s", d.Group, d.Name, d.Version)
+	return strings.Join(lo.Filter([]string{d.Group, d.Name, d.Version}, func(s string, _ int) bool { return s != "" }), ":")
```

Comment:
bugfix for packages without version defined (they are acceptable by package manager)